ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -24,7 +25,8 @@ class DeploymentUI(PageNavigator):
     def __init__(self, driver):
         super().__init__(driver)
         self.dep_loc = locators[self.ocp_version]["deployment"]
-        self.validation_loc = locators[self.ocp_version]["validation"]
+        if Version.coerce(self.ocp_version) <= Version.coerce("4.9"):
```

Comment:
Can you please use approach from version module I added?
See examples here:
https://github.com/red-hat-storage/ocs-ci/pull/4872/files

See email:
https://mailman-int.corp.redhat.com/archives/ocs-qe/2021-September/msg00292.html

Reply:
done